posts/zzz_DO_NOT_EDIT_rtflite_1.0.0/rtflite.qmd

```diff
@@ -0,0 +1,97 @@
+---
+title: 'rtflite 1.0.0: Production-Ready Clinical TLFs in Python'
+author:
+- name: Yilong Zhang
+- name: Nan Xiao
+description: Do you need to deliver and submit clinical study reports using Python? We have a solution.
+date: '2025-08-11'
+categories:
+- TLG
+image: logo.png
+---
+
+<!--------------- typical setup ----------------->
+
+```{r setup, include=FALSE}
+long_slug <- "2025-08-11_rtflite_1.0.0..."
+library(link)
+link::auto()
+```
+
+<!--------------- post begins here ----------------->
+
+We are thrilled to announce the release of rtflite 1.0.0,
+marking a significant milestone in bringing production-ready
+TLF generation capabilities in RTF format to Python for clinical trial reporting.
+This major release represents our commitment to providing the pharmaceutical
+industry with robust, reliable tools for creating tables, listings, and
+figures (TLFs) in RTF format.
+
+## What is rtflite?
+
+[rtflite](https://pharmaverse.github.io/rtflite/) is a Python package
```

Comment:
Can use auto-linking for anything not in a title - just type {rtflite} (no backticks)
```suggestion
{rtflite} is a Python package
```